docs/install/README.md

```diff
@@ -0,0 +1,422 @@
+
+
+Check that there are two Deployments. The output should look similar to this:
+
+```
+kubectl get deployment
+NAME                 READY   UP-TO-DATE   AVAILABLE   AGE
+cass-operator        1/1     1            1           2m
+k8ssandra-operator   1/1     1            1           2m
+```
+### Install into a different namespace
+This slight variation demonstrates how to install the operator into a different namespace.
+
+Create a namespace named `k8ssandra-operator`:
+
+```sh
+kubectl create ns k8ssandra-operator
+```
+Create a kustomization directory:
+
+```sh
+K8SSANDRA_OPERATOR_HOME=$(mktemp -d)
+cat <<EOF >$K8SSANDRA_OPERATOR_HOME/kustomization.yaml
+namespace: k8ssandra-operator
+
+resources:
+- github.com/k8ssandra/k8ssandra-operator/config/default?ref=main
+
+images:
+- name: k8ssandra/k8ssandra-operator
+  newTag: 7a2d65bb
+EOF
+```
+
+Note that the `namespace` property has been added. This property tells Kustomize to apply a transformation on all resources that specify a namespace.
+
+Now install the operator:
+
+```
+kustomize build $K8SSANDRA_OPERATOR_HOME | kubectl apply -f -
+```
+This installs the operator in `k8ssandra-operator`.
+
+## Deploy a K8ssandraCluster
+Now we will deploy a K8ssandraCluster that consists of a 3-node Cassandra cluster and a Stargate node.
+
+```sh
+cat <<EOF | kubectl apply -f -
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: demo
+spec:
+  cassandra:
+    cluster: demo
+    serverVersion: "4.0.0"
+    datacenters:
+      - metadata:
+          name: dc1
+        size: 3
+        storageConfig:
+          cassandraDataVolumeClaimSpec:
+            storageClassName: standard
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: 5Gi
+        config:
+          jvmOptions:
+            heapSize: 512M
+        stargate:
+          size: 1
+          heapSize: 256M
+EOF 
+```
+## Multi-cluster Install with Kustomize
+If you previously created a cluster with `setup-kind-multicluster.sh` we need to delete it in order to create the multi-cluster setup. The script currently does not support adding clusters to an existing setup (see [#128](https://github.com/k8ssandra/k8ssandra-operator/issues/128)).
+
+We will create two kind clusters with 3 worker nodes per clusters. Remember that K8ssandra Operator requires clusters to have routable pod IPs. kind clusters by default will run on the same Docker network which means that they will have routable IPs.
+
+### Create kind clusters
+Run `setup-kind-multicluster.sh` as follows:
+
+```
+./setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
+```
+
+When creating a cluster, kind generates a kubeconfig with the address of the api server set to localhost. We need a kubeconfig that has the api server address set to its internal ip address. `setup-kind-multi-cluster.sh` takes care of this for us. Generated files are written into a `build` directory.
```

Comment:
This isn't working as I'd expect. I run `setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4`. `docker ps` yields 
```
CONTAINER ID   IMAGE                           COMMAND                  CREATED              STATUS              PORTS                                                                                                     NAMES
ae8556d7b4da   kindest/node:v1.21.2            "/usr/local/bin/entr…"   About a minute ago   Up About a minute                                                                                                             k8ssandra-0-worker
7593b4cef241   kindest/node:v1.21.2            "/usr/local/bin/entr…"   About a minute ago   Up About a minute                                                                                                             k8ssandra-0-worker3
ce26eb5242bb   kindest/node:v1.21.2            "/usr/local/bin/entr…"   About a minute ago   Up About a minute   0.0.0.0:30080->30080/tcp, 0.0.0.0:30090->30090/tcp, 0.0.0.0:30942->30942/tcp, 127.0.0.1:49847->6443/tcp   k8ssandra-0-control-plane
7d64a4077f0c   kindest/node:v1.21.2            "/usr/local/bin/entr…"   About a minute ago   Up About a minute                                                                                                             k8ssandra-0-worker2
7ae2889b9dc1   moby/buildkit:buildx-stable-1   "buildkitd"              3 days ago           Up 3 days                                                                                                                     buildx_buildkit_condescending_wilson0
9bd36b3a1f37   registry:2                      "/entrypoint.sh /etc…"   3 weeks ago          Up 3 days           0.0.0.0:5000->5000/tcp, :::5000->5000/tcp                                                                 kind-registry
```

Looks like one cluster to me?

Reply:
What are you expecting to see with `docker ps`? Yes, kind will start containers, but is that a good way to check if the clusters are up?

Reply:
I run: `setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4`

I get back:

```
Creating clusters
Creating cluster "k8ssandra-0" ...
 ✓ Ensuring node image (kindest/node:v1.21.2) 🖼 
 ✓ Preparing nodes 📦 📦 📦 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✓ Joining worker nodes 🚜 
Set kubectl context to "kind-k8ssandra-0"
You can now use your cluster with:

kubectl cluster-info --context kind-k8ssandra-0

Thanks for using kind! 😊
Error response from daemon: endpoint with name kind-registry already exists in network kind

Generating kubeconfig
Generating in-cluster kubeconfig
```